libs/langgraph/langgraph/pregel/debug.py

```diff
@@ -72,7 +72,6 @@ def map_debug_tasks(
 
         metadata = config["metadata"].copy()
         metadata.pop("checkpoint_id", None)
-        metadata.pop("checkpoint_ns", None)
```

Comment:
so you remove it in one function but not the other?

Reply:
discussed offline -- bug in core, will update when fixed!